subprojects/docs/src/docs/userguide/jvm/groovy_plugin.adoc

```diff
@@ -281,7 +281,7 @@ include::sample[dir="snippets/groovy/compilationAvoidance/kotlin/ast-transformat
 Since 5.6, Gradle introduces an experimental incremental Groovy compiler. To enable incremental compilation for Groovy, you need:
 
 * Enable <<groovy_plugin.adoc#sec:groovy_compilation_avoidance,Groovy compilation avoidance>>.
-* Explicitly enable incremental Groovy compilation in the build script:
+* Explicitly enable incremental Groovy compilation and incremental compilation after a failure in the build script:
```

Comment:
This sentence is a little hard to read in the new state -- can we leave it as-is? Seems to me like incremental compilation includes "after a failure" by default now, so we don't need to call it out explicitly.

Reply:
Sounds good to me, reverted that change.